### PR TITLE
Mergin the 10Clouds addition after the Kudelski audit code freeze.

### PR DESCRIPTION
**Pull request by jalooc**

### CODE REVIEW

**Comment by jalooc** [comment]
Dang, I can't approve my own PR, even though in this exceptional case I should be able to :P

I clicked through the updated extension and scanned through the changes - all looks good (apart of some code nitpicks, but this is not a place for those), all seems to be working fine.

**Comment by jalooc** [comment]
Hmm that's weird - the same commit hash [passed the pipeline when it was being PRed last time](https://github.com/Cardinal-Cryptography/aleph-zero-signer/actions/runs/4511894828) 🧐

**Comment by jalooc** [comment]
...fixed with [this](https://github.com/Cardinal-Cryptography/aleph-zero-signer/pull/58/commits/d340f67a9c43cda2d6f05124f81b6cadd6d790b0) by trial and error. Not sure why normal assignemnt was not working and `Object.assign` is.

**Comment by youPickItUp** [comment]
> ...fixed with [this](https://github.com/Cardinal-Cryptography/aleph-zero-signer/pull/58/commits/d340f67a9c43cda2d6f05124f81b6cadd6d790b0) by trial and error. Not sure why normal assignemnt was not working and `Object.assign` is.

`crypto` is suppose to be read only, but it's properties not:
https://developer.mozilla.org/en-US/docs/Web/API/crypto_property
Not sure how it works :sweat_smile:

File: packages/extension-ui/src/Popup/Signing/index.tsx [hunk 1]
```diff
@@ -76,14 +83,24 @@ export default React.memo(
   styled(Signing)(
     ({ theme }: Props) => `
     .content {
-      outline: ${theme.newTransactionBackground} solid 37px;
+      // outline: ${theme.newTransactionBackground} solid 37px;
```
Comment (youPickItUp) [inline packages/extension-ui/src/Popup/Signing/index.tsx:86]: I'd remove this

**Review by youPickItUp — APPROVED** [review]

**Pull request merged**